tests/org.jboss.tools.freemarker.ui.bot.test/src/org/jboss/tools/freemarker/ui/bot/test/FreemarkerTest.java

```diff
@@ -263,11 +265,7 @@ public void emptyErrorLog() {
 		
 		LogView elv = new LogView();
 		elv.open();
-		try {
-			new ContextMenu("Delete Log").select();
-		} catch (RedDeerException e) {
-			// do nothing
-		}
+		elv.clearLog();
```

Comment:
clear log is different action than delete log. Are you sure you want to just clear it ?